prime-router/src/main/kotlin/azure/ActionHistory.kt

```diff
@@ -616,6 +624,45 @@ class ActionHistory(
             )
         }
 
+        val lineages = Report.createItemLineagesFromDb(header, sentReportId)
+        lineages?.forEach { itemLineage ->
```

Comment:
Makes sense to me.  My ultimate preference would to have a clearer delineation of the uses cases since this doesn't even feel like monitoring and is more an analytics/business reporting question